Gems/OpenParticleSystem/Code/Include/OpenParticleSystem/ParticleRequestBus.h

```diff
@@ -8,33 +8,45 @@
  
 #pragma once
 
-#include <AzCore/Asset/AssetCommon.h>
-#include <AzCore/Component/EntityId.h>
-#include <AzCore/EBus/EBus.h>
-#include <OpenParticleSystem/Asset/ParticleAsset.h>
+#include <AzCore/Component/ComponentBus.h>
 
 namespace OpenParticle
 {
     class ParticleRequest
-        : public AZ::EBusTraits
+        : public AZ::ComponentBus
     {
     public:
-        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
-        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
-        using BusIdType = AZ::EntityId;
-
+        /**
+         * @brief Triggers the particle system to start playing.
+         */
```

Comment:
Nit, https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md suggests just univerally using doxygen format, which is described in that doc.
```cpp
//! First line of description, until the first period is encountered.
//! Any additional information, after the first period.
//! (Optional) @param paramname describe the param until the first period.  Additional
//! information after the first period, if necessary.
```